rioting-daisies/BangazonWorkforce
# Added Readme, Added Sql file to create DB, Assigning Employees to Training

# Description

I've created a Readme for the project.

Added the BangazonAPI.sql file for creating the database on your local machine.

Implemented the functionality for assigning an employee to a training program.



Fixes Issue # (10)[https://github.com/rioting-daisies/BangazonWorkforce/issues/10]

## Type of change






# Testing Instructions

1. `git fetch --all`
2. `git checkout blue-AssignTraining`
3. Run the application
4. Navigate to the Employees page, and click on the details on an employee.
5. Click on the link to assign the employee to a training program.
6. Assign the employee, and confirm that the training program is added to the list.
7. Review the code in `EmployeesController.cs`, `AssignTrainingProgramViewModel.cs`



# Checklist:
